src/React.purs

```diff
@@ -294,6 +300,26 @@ foreign import getRefs :: forall props state access eff.
   ReactThis props state ->
   Eff (refs :: ReactRefs (read :: Read | access) | eff) Refs
 
+foreign import data Ref :: Type
+
+-- | Read named ref from Refs
+foreign import readRefImpl :: forall props state access eff.
+  ReactThis props state ->
+  String ->
+  Eff (refs :: ReactRefs (read :: Read | access) | eff) (Nullable Ref)
+
+readRef :: forall props state access eff.
```

Comment:
Could you please add comments to these as well?

Reply:
`Ref` type should also be documented, and advising the user to use `unsafeCoerce` make me feel I am doing something wrong, so either we provide a third library with `Node -> Ref` and `Ref -> Node` safe functions (`purescript-react-refs`) or we can include that in `purescript-react-dom`, which already depends on `purescript-dom`. 
 There are pros and cons of both, but either will provide a safe way for the basic operation.

By the way: using zypher & webpack (with uglify plugin) on [purescript-isomorphic-react-example](https://github.com/coot/purescript-isomorphic-react-examples), which is pulling DOM  for very basic stuff (access to location, history, search the mounting point in the DOM), pulls ~2.3KB of data from `purescript-dom`, which is ~0.5% of whole bundle (~400KB, which gzips to 94KB). zypher, to some extend can remove things from foreign modules that's why the overall code from DOM that lands in the bundle is pretty minimal.

Reply:
In my opinion, I think `purescript-react-dom` would be a nice spot for functions that convert between `Ref` and `Node`. Similarly, in a `purescript-react-native` library, there could be functions that convert between `Ref` and something native.